docs/index.mdx

```diff
@@ -28,7 +28,22 @@ import { RegionalUrl } from "@site/src/components/RegionalUrls";
 
 **LangSmith** is a platform for building production-grade LLM applications.
 It allows you to closely monitor and evaluate your application, so you can ship quickly and with confidence.
-LangChain's open source frameworks [langchain](https://python.langchain.com) and [langgraph](https://langchain-ai.github.io/langgraph/) work seemlessly with LangSmith but are not necessary - LangSmith works on its own!
+With LangSmith you can:
+- **Trace LLM Calls**: Gain visibility into LLM calls, and other parts of your application's logic. 
+- **Evaluate Performance**: Compare results across models and prompts to identify what works best.
+- **Improve Prompts**: Quickly refine prompts to achieve more accurate and reliable results.
+
+:::tip LangSmith + LangChain OSS
+
+LangSmith integrates seamlessly with LangChain's open source frameworks [LangChain](https://python.langchain.com) and [LangGraph](https://langchain-ai.github.io/langgraph/), with no extra instrumentation needed. 
```

Comment:
nit: imo clearer that the second "langchain" is referring to a package if it's lowercased
```suggestion
LangSmith integrates seamlessly with LangChain's open source frameworks [langchain](https://python.langchain.com) and [langgraph](https://langchain-ai.github.io/langgraph/), with no extra instrumentation needed. 
```

Reply:
Yeah i get what you're saying, kinda weird because don't use lowercase langchain/langgraph anywhere else in the docs. Gonna make the formatting a codeblock - `langchain`/`langgraph` which makes sense since we're referencing the name of the packages